repos/system_upgrade/el7toel8/actors/dhclientreport/actor.py

```diff
@@ -0,0 +1,36 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.libraries.common.rpms import has_package
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.reporting import Report, create_report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckDhclientActor(Actor):
+    """
+    Check if dhclient is installed
+    and report backward incompatible change in CLI options.
+    """
+
+    name = "dhclient"
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        if not has_package(InstalledRedHatSignedRPM, 'dhclient'):
```

Comment:
@landgraf In case this is related just for RHEL 7.6:
```suggestion
        if not has_package(InstalledRedHatSignedRPM, 'dhclient') and version.matches_source_version(['7.6']):
```

and add import:
```
from leapp.libraries.common.config import version
```

Reply:
but in such case, you would need modify even tests and move the code to the library, to be able to mock the config object.

Reply:
@pirat89  Thank you for the comment. It's not just for 7.6 but for 7.6+